rllib/agents/trainer.py

```diff
@@ -1053,7 +1053,9 @@ def _validate_config(config: PartialTrainerConfigDict):
             raise ValueError("`model._time_major` only supported "
                              "iff `_use_trajectory_view_api` is True!")
 
-        if type(config["input_evaluation"]) != list:
+        if type(config["input_evaluation"]) == tuple:
+            config["input_evaluation"] = list(config["input_evaluation"])
+        elif type(config["input_evaluation"]) != list:
```

Comment:
```suggestion
        if isinstance(config["input_evaluation"], tuple):
            config["input_evaluation"] = list(config["input_evaluation"])
        elif not isinstance(config["input_evaluation"], list):
```

(I know this isn't present in the original code but I think we should make this change now just to be safe)

Reply:
Yes, you are right it's cleaner.
I wanted to do the same, but in the meantime, I also wanted to be close to the original code :blush: